trade-module/src/main/java/io/edpn/backend/trade/application/service/RequestSystemCoordinatesService.java

```diff
@@ -26,9 +26,9 @@ public boolean isApplicable(System system) {
 
     @Override
     public void request(System system) {
-        SystemDataRequest stationDataRequest = new SystemDataRequest();
-        stationDataRequest.setSystemName(system.getName());
-
+        SystemDataRequest stationDataRequest = new SystemDataRequest(
+                null, system.getName()
```

Comment:
requesting module is empty, is this set somewhere else?

ohterwise this must be "trade" here